openc3/lib/openc3/packets/packet.rb

```diff
@@ -1032,6 +1112,22 @@ def self.from_json(hash)
       packet
     end
 
+    def decom
+      # Read all the RAW at once because this could be optimized by the accessor
+      json_hash = read_items(@sorted_items)
```

Comment:
So read_items initializes the hash and then we call read_item again but pass given_raw to really populate it? I'm confused about the 2 pass system here.